<launch>
    <node pkg="pkg1" type="zed_camera.py" name="camera" respawn="false" output="screen"/>
    <node pkg="rviz" type="rviz" name="rviz"/>
    <node pkg="pkg1" type="zed_camera_receiver.py" name="receiver" respawn="false" output="screen"/>
    <param name="model_name" type="str" value="ZED_CG3"/>
    <param name="target_epoch" type="int" value="-1"/>
    <param name="image_height" type="int" value="720"/>
    <param name="image_width" type="int" value="1280"/>
</launch>